GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -107,6 +107,16 @@ internal Keys GetShortcutKeys(Commands cmd)
 
         #endregion
 
+        public void RevisionDiff_KeyDown(object sender, KeyEventArgs e)
```

Comment:
It is not a very good idea to expose event handlers - this is breaking encapsulation and leaking internal details. This also allows external components couple to this, thus making it impossible to replace the component, if need arises.
A better way would be create a public API, say, `bool Handle(Keys)` which then can be invoked by this handler and by external consumers.